### DISCUSSION TITLE
Color coded prompt texts

### DISCUSSION THREAD

**Discussion by mocmocmoc** [Nevysha/Cozy-Nest#54] [Ideas]
Possible to have different custom colors for prompt 
(emphasis), [de-emphasis], <LORA embeddings>, <TI embeddings>, etc?

Maybe even brighter/darker shades depending on multiplier strength.

**Comment by Nevysha** [comment]
that's in my todo list ! which can be found [here](https://exclusive-drink-8c5.notion.site/Nevysha-Cozy-Nest-f95f333908f0406f990ed603b424780c) . It's messy because that's literaly my personal quick note.
